verilog/tools/formatter/verilog_format.cc

```diff
@@ -60,6 +60,9 @@ ABSL_FLAG(int, show_largest_token_partitions, 0,
 ABSL_FLAG(bool, show_token_partition_tree, false,
           "If true, print diagnostics after token partitioning and then "
           "exit without formatting output.");
+ABSL_FLAG(bool, show_token_partition_policy, false,
+          "If true, print diagnostics after unwrapping token parition tree and then "
+          "exit without formatting output.");
```

Comment:
Could we re-use `show_token_partition_tree`.  It turns out I have a TODO to add this information in the existing partition tree dump.

Reply:
Added to `show_token_partition_tree`